shared/src/vyxal/Elements.scala

```diff
@@ -1010,10 +1010,12 @@ object Elements:
       false,
       "a: lst -> [a[0], a[-1]]",
       "a: str -> [a[0], a[-1]]",
```

Comment:
You're adding new behavior, so be sure to add an overload here, something like `"a: num -> blahb lah"`

Reply:
fixed this, potential documentation standards for complex numbers?

Reply:
I can't think of anything specific. Maybe `[a.real, a.imag]`?